native/SampleApps/RestAPIExplorer/RestAPIExplorer.xcodeproj/project.pbxproj

```diff
@@ -944,6 +944,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_LAUNCHIMAGE_NAME = LaunchImage;
 				BUNDLE_PREFIX = com.salesforce.mobilesdk;
+				CLANG_ENABLE_MODULES = YES;
```

Comment:
I thought we tried hard getting rid of CLANG_ENABLE_MODULES.
Why is it needed again?